content/en/agent/guide/datadog-disaster-recovery.md

```diff
@@ -0,0 +1,284 @@
+---
+title: Datadog Disater Recovery
+private: true
+further_reading:
+- link: "agent/remote_config/?tab=configurationyamlfile"
+  tag: "Documentation"
+  text: "Remote Configuration"
+- link: "/getting_started/site/"
+  tag: "Documentation"
+  text: "Getting Started with Datadog Sites"
+---
+
+{{< callout url="https://www.datadoghq.com/product-preview/datadog-disaster-recovery/" d-toggle="modal" d_target="#signupModal" custom_class="sign-up-trigger" >}}
+Datadog Disaster Recovery is in Preview, but you can request access! Use this form to submit your request.
+{{< /callout >}}
+
+## Overview 
+Datadog Disaster Recovery (DDR) provides you with observability continuity during events that may impact a cloud service provider region or Datadog services running within a cloud provider region. Using DDR, you can recover live observability at an alternate, functional Datadog site in typically under two hours, enabling you to meet your critical observability availability goals.
+
+DDR also allows you to periodically conduct disaster recovery drills to not only test your ability to recover from outage events but to also meet your business and regulatory compliance needs.
+
+
+## Prerequisites 
+Datadog Agents versions **7.54 or above** is required for Datadog Disaster Recovery. 
+
+#### Supported telemetry types and products
+The Agent-based failover description provided on this page supports failover of metrics, traces, and logs telemetry types. It also supports Datadog Infrastructure Monitoring, APM, and Logs products.
+
+<div class="alert alert-info">
+Datadog is continuously evaluating customer requests to support DDR for additional products. Contact the [Disaster Recovery team](mailto:disaster-recovery@datadoghq.com) to learn about upcoming capabilities and your specific needs if they are not covered above.
+</div>
+
+
+## Setup 
+To enable Datadog Disaster Recovery, follow the relevant steps for when:
+
+1. [you are ready to configure Datadog Disaster Recovery](#when-you-are-ready-to-configure-datadog-disaster-recovery)
+2. [Datadog confirms your new organization as your secondary failover organization](#when-datadog-confirms-your-new-organization-as-your-secondary-failover-organization)
+3. [you have linked the DDR org to your primary org](#when-you-have-linked-the-ddr-org-to-your-primary-org)
+4. [you are ready to test the failover process](#when-you-are-ready-to-test-the-failover-process)
+
+#### when you are ready to configure Datadog Disaster Recovery: 
+{{% collapse-content title=" 1. Create your secondary Datadog organization" level="h5" %}}
```

Comment:
I notice that "secondary" and "DDR" seem to be kind of interchangeable throughout this topic, which feels a bit confusing. Can you pick one and stick with it so your user doesn't have to make the equivalency in their head?